apps/studio/data/custom-domains/custom-domains-query.ts

```diff
@@ -105,9 +92,9 @@ export async function getCustomDomains(
       } as const
     }
 
-    const isNoHostnameConfiguredError =
-      (response.error as any)?.code === 400 &&
-      (response.error as any)?.message?.includes('custom hostname configuration')
+    const isNoHostnameConfiguredError = (error as any)?.message?.includes(
```

Comment:
probably type CustomDomainsError as ResponseError on line 113, then won't need the `any` here

Reply:
Good shout!